customer-commons/src/main/java/no/unit/nva/customer/ObjectMapperConfig.java

```diff
@@ -0,0 +1,16 @@
+package no.unit.nva.customer;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import nva.commons.utils.JsonUtils;
+
+public class ObjectMapperConfig {
+
+    public static final ObjectMapper objectMapper;
+
+    static {
+        objectMapper = JsonUtils.jsonParser;
+        objectMapper.registerModule(new JavaTimeModule());
```

Comment:
Should this be added to nva-commons ?

Reply:
Actually it is there but commented out
`            //.registerModule(new JavaTimeModule()) JavaTimeModule belongs to an obsolete library`